ROADMAP.md

```diff
@@ -0,0 +1,359 @@
+- A fully working example (ping test) used in go-libp2p and rust-libp2p CIs,
+- An interoperability Dashboard that shows how implementations and versions are tested.
+
+### 2. Refresh Testground's EKS support
+
+- Delivery: Q4 2022
+- Theme: usefulness
+- Effort: approx. 6 months
+
+**Why:** Testground can simulate small networks in CI, but it covers more use cases when it lives in a larger cluster. When we run Testgroun in Kubernetes, we can support whole organizations through the Testground As A Service product.
+
+Using a managed service (Amazon's Elastic Kubernetes Service) means our maintenance costs are lower, and the team can focus on improvements.
+
+**Deliverables:**
+
+- An EKS installation script,
+  - Extra care is taken on Network infrastructure (CNIs).
+- A (fixed) Kubernetes runner that runs on Amazon's EKS,
+- The team can use the latest Testground version, create new releases, and upgrade the cluster.
+
+### 3. Improve our Development & Testing infrastructure to meet engineering standards
+
+- Delivery: Q1 2023
+- Theme: reliability & sustainability
+- Effort: approx. 4 months
+
+**Why:** Testground proved itself valuable multiple times over the years. However, now we need bulletproof development processes to make the project sustainable and facilitate external contributions.
+
+Extra care is taken on Testing and Stability: we are building a testing platform, and Testground's testing must be impeccable.
+
+**Deliverables:**
+
+- Automated release process with official releases, binary distribution, and Changelogs - [EPIC 1430](https://github.com/testground/testground/issues/1430)
+- Well-defined Project management processes like triaging, contribution & reviewing processes, etc.
+- Documentations for maintainers,
+- Maintainable integration testing tooling (no more shell scripts or flakiness),
+- A Stability Dashboard used to identify regression & discuss improvement with Maintainers and Users,
+- Tooling for EKS testing.
+
+### 4. Provide a Testground As A Service Cluster used by libp2p & ipfs teams
+
+- Delivery: Q3 2023
+- Theme: usefulness
+- Effort: approx. 4 months
+
+**Why:** Improving experience with faster testing in user's CI. Cover more use cases with large-scale networks.
+
+**Deliverables:**
+
+- A _stable_ cluster,
+- Authentication,
+- Tooling for users to use EKS cluster in their testing,
+- Integration of the EKS feature in our testing infrastructure
+  - Short-lived clusters used during integration testing
+
+### 5. Testground Is Usable by Non-Testground experts
+
+- Delivery: Q4 2023
+- Theme: sustainability
+- Effort: approx. 8 months
+
+**Why:** Testground is an Open Source project, and the more project that uses it, the more improvement we'll see. We want to drive adoption outside of Protocol Labs but also encourage contribution. Other organizations have picked up the tool and started contributing; we want to lower friction and strike the Iron while it's hot.
```

Comment:
```suggestion
**Why:** Testground is an Open Source project, and the more projects that use it there are, the more external contributions we'll see. We want to drive adoption outside of Protocol Labs and encourage contributions. Other organizations have picked up the tool and started contributing; we want to lower the friction and strike the iron while it's hot.
```

I also feel like this is not quite capturing `Testground Is Usable by Non-Testground experts`. It seems to be more about external adoption and contributions rather than being beginner friendly.

Reply:
> If we attract more people to use Testground, we'll have more contributions to the project because Testground is an Open Source project.